packages/hardhat-test-helpers/src/chai/types.ts

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line @typescript-eslint/no-namespace, @typescript-eslint/no-unused-vars
+declare namespace Chai {
```

Comment:
The existing Chai types specify that these comparison operators should only accept `number` (or `number|Date` for some of them). Without these changes, we can't override the methods to support additional types. Here's the actual error output that occurs: https://gist.github.com/feuGeneA/4e09115228b205887a41d981c2409de7

Since you've got me thinking about it, though, I'm wondering if we should specify here which types we're overloading it for, rather than just saying `any`. What do you think about that?